index.html

```diff
@@ -17,29 +17,37 @@
 <body>
     <header>
         <nav class="navbar">
+          <div class="logo" href="#">Maryam</div>
           <ul class="navbar-flex">
-            <li class="nav-logo"><a href="#">Maryam</a></li>
-            <li class="nav-logo">
+            <li class="nav-link"><a href="#">Portfolio</a></li>
+            <li class="nav-link"><a href="#">About</a></li>
+            <li class="nav-link"><a href="#">Contact</a></li>
+            <li class="envelope"><a href="#"></a></li>
+            <li class="mob-bars">
               <a href="#"><i class="fa-solid fa-bars"></i></a>
             </li>
           </ul>
         </nav>
       </header>
       
       <section class="main-div">
-        <h1 class="main-heading">Hello There. I'm Maryam</h1>
+        <div class="intro-div">
+      <h1 class="main-heading">Hey There. <br> I'm Maryam.</h1>
         <h3 class="support-text">I'am a Software Developer</h3>
         <p class="descriptive-text">
           I can help you build a product , feature or website Look through some of
           my work and experience! If you like what you see and have a project you
           need coded, don't hesitate to contact me.
         </p>
+        </div>
+        <div class="accounts-flex">
         <ul class="accounts">
           <li>
             <a href="#"><img src="/images/github.png" alt=""></a>
           </li>
           <li>
-            <a href="#"><img src="/images/linkedin.png" alt=""></a>
+          <a href="#"><img src="/images/linkedin.png" alt=""></a>
+
           </li>
```

Comment:
- [ ] You did a nice job however,   the social media icons are not displayed on the desktop version as per the Figma design.

Your design
![port3](https://user-images.githubusercontent.com/53347966/229103847-199175a6-dcbb-4354-9735-eb9ebe2b016d.JPG)

Figma design
![figma](https://user-images.githubusercontent.com/53347966/229104070-78f41376-c70a-4efc-83fc-ccf487a1b090.JPG)

Reply:
@algerina They are not showing because I have used images for that. Its visible in my live version. What should I do now?

Reply:
@maryam0007 You can fix the issue by adding a`.` at the beginning of every image link. I should be something like `<img src="./images/github.png" alt="">`.